packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1720,13 +1720,33 @@ class Flexible extends ParentDataWidget<Flex> {
 
 /// A paragraph of rich text.
 ///
-/// Consider using [Text], which integrates with [DefaultTextStyle], rather than
-/// this widget, which requires explicit styling.
+/// The [RichText] widget displays text using a variety of different styles. The
+/// text to display is described using a tree of [TextSpan] objects, each of
+/// which has an associated style, which is used for that subtree. The text
+/// might break across multiple lines or might all be displayed on the layout
+/// constraints.
+///
+/// Text displayed in a [RichText] widget must be explicitly styled. When
+/// picking which style to use, consider using the closest enclosing
+/// [DefaultTextStyle] to provide defaults.
```

Comment:
mention [DefaultTextStyle.of] so they don't go and try to walk the tree to find the nearest one or something